tree/dataframe/inc/ROOT/RDataFrame.hxx

```diff
@@ -47,6 +48,7 @@ public:
    RDataFrame(TTree &tree, const ColumnNames_t &defaultBranches = {});
    RDataFrame(ULong64_t numEntries);
    RDataFrame(std::unique_ptr<ROOT::RDF::RDataSource>, const ColumnNames_t &defaultBranches = {});
+   RDataFrame(ROOT::Internal::RDF::RDatasetSpec spec);
```

Comment:
Let's hide this constructor from the documentation, follow this example https://github.com/root-project/root/blob/b3bd014f47c0f44849f19a66ed7bdc83506a03b4/tree/dataframe/inc/ROOT/RResultPtr.hxx#L136